# m.m00
float64 m00
# center of red area
geometry_msgs/Point center